inputfiles/browser.webidl.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
what is the source of this file?

Reply:
is that one you hand authored? I am asking mainly for naming purposes. i think browser.webidl is not a good name for this one..